### PR TITLE
Feat/showcase mode preparation

### PR DESCRIPTION
**Pull request by Mrkazik99**

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -318,51 +318,13 @@ webdav:
             {}
 
 redis:
-    enabled: true
-    auth:
-        enabled: false
-        
-    architecture: standalone
+    enabled: false
```
Comment (loboda4450) [inline values.yaml:321]: shouldnt all of that be true?

File: values.yaml [hunk 2]
```diff
@@ -318,51 +318,13 @@ webdav:
...
-        enabled: true
-        users:
-        - username: voltaserve
-          password: voltaserve
-          disabled: false
-          policies:
-          - readwrite
-          - consoleAdmin
-          - diagnostics
+    enabled: false
```
Comment (loboda4450) [inline values.yaml:324]: .

File: values.yaml [hunk 3]
```diff
@@ -318,51 +318,13 @@ webdav:
...
-            enabled: false
-    conf:
-        single-node: true
-    tls:
-        enabled: false
-    initJob:
-        dbUsername: test
-        dbPassword: test
-    
+    enabled: false
```
Comment (loboda4450) [inline values.yaml:327]: .

File: values.yaml [hunk 4]
```diff
@@ -318,51 +318,13 @@ webdav:
...
+
 meilisearch:
-    enabled: true
\ No newline at end of file
+    enabled: false
```
Comment (loboda4450) [inline values.yaml:330]: .

**Review by loboda4450 — CHANGES_REQUESTED** [review]

File: values.yaml (hunk 1 above)
Comment (Mrkazik99) [reply values.yaml:321]: Not in regular values file

File: values.yaml (hunk 2 above)
Comment (Mrkazik99) [reply values.yaml:324]: .

File: values.yaml (hunk 3 above)
Comment (Mrkazik99) [reply values.yaml:327]: .

File: values.yaml (hunk 4 above)
Comment (Mrkazik99) [reply values.yaml:330]: .

File: values.yaml (hunk 1 above)
Comment (Mrkazik99) [reply values.yaml:321]: Please take a look on values-showcase.yaml

Comment (loboda4450) [reply values.yaml:321]: fuck, that blob made me skip the file change XD

**Review by loboda4450 — APPROVED** [review]

**Pull request merged**